tests/test_catalog.py

```diff
@@ -19,26 +19,31 @@
     LocalSource,
     RemoteSource,
 )
+from jschon.catalog import Source
 from jschon.vocabulary import Metaschema, Keyword
 from tests import example_schema, metaschema_uri_2020_12, core_vocab_uri_2020_12
 
 json_example = {"foo": "bar"}
 
 
 @pytest.fixture
-def local_catalog():
```

Comment:
There's no need to split up this fixture; `local_catalog` will work fine for the new test function. In any case it's worth making sure that the default source works irrespective of other registered sources.

Reply:
I remember having to do this for some reason, but I'll investigate.  If I can't remember what it was then I'll un-split them.

Reply:
I couldn't figure out why I split out the empty catalog, so I reverted it.  I vaguely think I needed it for some other PR as well, but if so I'll handle it then (and it might have been a PR I abandoned).